features/facebookImport/src/popUps/infoPopUps/offFacebookInfo.jsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+
+import i18n from "../../i18n.js";
+import BaseInfoPopUp from "../baseInfoPopUp/baseInfoPopUp.jsx";
+import Infographic from "../../components/infographic/infographic.jsx";
+
+const OffFacebookInfoScreen = () => {
```

Comment:
This is very similar to others. Is there no way to factor the common boilerplate out?

Reply:
That is the next step. Have one infoPopUp and let it be populated by data. Nothing for this PR though

Reply:
Given that most of the common code is being imported from BaseInfoPopUp I don't think that there much more code saving that can be done here